package.json

```diff
@@ -25,7 +25,7 @@
     "test-typescript": "karma start ./test/deployment/typescript/karma.conf.js",
     "test:ci": "cross-env SHOW_DIFF=true npm run test",
     "createreferences": "node test/common/reference-server.js",
-    "prettier": "prettier --write {src,tests,typings}/**/*.{ts,js}",
+    "prettier": "prettier --write {playground,src,tests,typings}/**/*.{ts,js}",
     "lint": "eslint {src,tests,typings}/**/*.{ts,js}"
```

Comment:
Sorry for nitpicking, but we should also add it to the `lint` task then.

Reply:
Makes sense, I updated that 👌